example/lib/nested_realtime_events.dart

```diff
@@ -21,7 +21,7 @@ void listenRealtimeConnection(ably.Realtime realtime) {
 
   //DISPOSE ON CONNECTED
   final stream = realtime.connection.on();
-  StreamSubscription omegaSubscription;
+  late StreamSubscription omegaSubscription;
   omegaSubscription = stream.listen((stateChange) async {
```

Comment:
This can be `final omegaSubscription = ...`, and the line above can be removed? Same for the other subscriptions.

Interestingly this file is not being used (as mentioned at the top of the file). Is it still valid to keep?

Reply:
We're using `await omegaSubscription.cancel();` inside the `onData` callback of `listen` which won't recognize the `omegaSubscription` if it is not already defined/declared.

I think we can remove this once we have a more modular example app.